operator/src/main/java/org/bf2/operator/managers/OperandOverrideManager.java

```diff
@@ -41,6 +45,14 @@ public void setImage(String image) {
             this.image = image;
         }
 
+        public List<EnvVar> getEnv() {
+            return env;
+        }
+
+        public void setEnv(List<EnvVar> env) {
```

Comment:
I'm wondering if it would be nicer to work with if the api was more declarative:

```suggestion
        public void overriderEnvVar(String name, String value) {
```
and or

```suggestion
        public void overriderEnvVar(EnvVar override) {
```

It would give us a reasonable place to hook in validation for duplicate overrides or to have a allow/deny list of variables (I'm happy with the idea of trusting it will be used wisely) .

Reply:
I struggled to get Jackson to accept this.  Feel free to push a commit or guide me with a response.

Reply:
It would probably end up being some nasty combination of `@JsonIgnore` and `@JsonSetter` for a default visible setter. Probably not worth the effort.

Reply:
Ok, let's leave it.